src/components/CategoriesPage.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line
```

Comment:
Kindly remove these disabled Linters, Linting helps in writing clean and error-free code ✊ 

Please check for other occurrences 👍

Reply:
Fixed required changes